src/transpose/index.js

```diff
@@ -4,6 +4,15 @@
  * @param {number[]} array The array to transpose
  * @returns {number[]} The transposed array
  */
-function transpose(array) {}
+function transpose(array) {
+  var trans = new Array (array[0].length);
+  for (var i = 0; i < trans.length; i++){
+      trans[i] = new Array(array.length);
```

Comment:
Same here. You can look into [Array.from](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) and [Array.fill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill)